### DISCUSSION TITLE
Would PipeCD apply to join CNCF Landscape?

### DISCUSSION THREAD

**Discussion by lloydchang** [pipe-cd/pipecd#2890] [Q&A] [answered]
Hi @nghialv 

_I am asking out of curiosity._

❔  Would PipeCD apply to join CNCF Landscape?

From https://github.com/cncf/landscape#new-entries
> **New Entries**
> Cloud native projects with at least 300 GitHub stars that clearly fit in an existing category are generally included. Put the project in the single category where it best fits.

I ask because [**#pipecd**](https://cloud-native.slack.com/archives/C01B27F9T0X) and [**#pipecd-jp**](https://cloud-native.slack.com/archives/C01BYNXAJKH) channels are in [**CNCF Slack**](https://cloud-native.slack.com/) at: 
• [**#pipecd** https://cloud-native.slack.com/archives/C01B27F9T0X](https://cloud-native.slack.com/archives/C01B27F9T0X)
• [**#pipecd-jp** https://cloud-native.slack.com/archives/C01BYNXAJKH](https://cloud-native.slack.com/archives/C01BYNXAJKH)

For example: Argo and Flux are CNCF Incubating projects
• https://landscape.cncf.io/card-mode?category=continuous-integration-delivery

Argo:
• https://blog.argoproj.io/argo-goes-to-cncf-incubator-f0e9dfb40597
• https://www.cncf.io/blog/2020/04/07/toc-welcomes-argo-into-the-cncf-incubator/
• https://www.cncf.io/projects/argo/

Flux:
• https://www.weave.works/blog/flux-joins-the-cncf-sandbox
• https://www.cncf.io/blog/2021/03/11/cncf-toc-votes-to-move-flux-from-sandbox-to-incubation/
• https://www.cncf.io/projects/flux/

Thank you.

**Comment by nghialv** [comment, marked as answer]
@lloydchang Thank you for your question.

Sorry for my late reply because I was missing the notification for this discussion.

> ❔ Would PipeCD apply to join CNCF Landscape?

Yes, definitely.
Tbh, we are on the way to preparing to apply.

**Reply by nghialv** [reply]
@lloydchang PipeCD has been accepted to join CNCF Landscape.
https://landscape.cncf.io/?selected=pipe-cd
Thank you for your nice suggestion. 💯
